DSharpPlus/Entities/Channel/Message/DiscordMessage.cs

```diff
@@ -391,7 +398,21 @@ internal DiscordMessageReference InternalBuildMessageReference()
             return reference;
         }
 
+        private IMention[] GetMentions()
+        {
+            var mentions = new List<IMention>();
+
+            if (this.ReferencedMessage != null && this._mentionedUsers.Contains(this.ReferencedMessage.Author))
+               mentions.Add(new RepliedUserMention()); // Return null to allow all mentions
```

Comment:
Do you intend for this to be the only thing that is added to `mentions` if it contains the author?

Reply:
No I don't think so? It worked in testing, but I'll look again

Reply:
Because the mentioneduser check will run right below it and add the user still.